proposals/0192-non-exhaustive-enums.md

```diff
@@ -4,6 +4,7 @@
 * Author: [Jordan Rose](https://github.com/jrose-apple)
 * Review Manager: [Ted Kremenek](https://github.com/tkremenek)
 * Status: **Implemented (Swift 5.0)**
+* Upcoming Feature Flag: `NonfrozenEnumExhaustivity` (implemented in Swift 6.0 to subsume bespoke compiler flag for Swift 4/4.2 language mode)
```

Comment:
```suggestion
* Upcoming Feature Flag: `NonfrozenEnumExhaustivity` (implemented in Swift 6.0) (enabled in Swift 6 language mode)
```

This should also include the `(enabled in Swift 6 language mode)` annotation to indicate when the flag is always enabled.

The details of what an upcoming feature flag does is not included as part of the header field. The details should be included in the "Source Compatibility" section.

Reply:
It is enabled in Swift 5 language mode too